### PR TITLE
Bump golang.org.x/crypto to the latest

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Bump golang.org.x/crypto to the latest to avoid
a low level security issue.

Addresses:  https://bugzilla.redhat.com/show_bug.cgi?id=1910255

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
OK, I'm confused, I'm not sure why varlink got bumped in some of the go files.  Anyway, I'll leave this over night to see if the tests are happy with the crypto changes upstream.

File: go.mod [hunk 1]
```diff
@@ -57,10 +57,11 @@ require (
 	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
 	github.com/uber/jaeger-client-go v2.25.0+incompatible
 	github.com/uber/jaeger-lib v2.2.0+incompatible // indirect
+	github.com/varlink/go v0.4.0
```
Comment (mheon) [inline go.mod:60]: You've re-added Varlink somehow - I think this needs a fresh revendoring

Comment (rhatdan) [reply go.mod:60]: Yes most likley you have some file in your go source that is no longer in master.  
grep -r github.com/varlink .

Should find the file causing this.

Comment (TomSweeneyRedHat) [reply go.mod:60]: Tried that last night and didn't spot anything.  I badly need to upgrade my sandbox VM, I think this is a good excuse to do so and try vendoring again in a fresh environment.  I'm going to close this one.

**Pull request closed without merging**